Framework/Core/Windows/Devices/Pwm/PwmEnums.cs

```diff
@@ -0,0 +1,8 @@
+namespace Windows.Devices.Pwm
+{
+    public enum PwmPulsePolarity
+    {
+        ActiveHigh = 0,

```

Comment:
The UWP enum is in this order, though typically "Low" is zero.  In this case, it's appropriate since "Active High" is the default.